data/110/875/899/9/1108758999.geojson

```diff
@@ -24,26 +27,30 @@
     "meso:admin_2":"Republika Srpska",
     "meso:gadm_adm2":"Banja Luka",
     "meso:gadm_adm3":"Dubica",
-    "meso:gaul_adm3":"Bosanska Dubica",
+    "meso:gaul_adm3":"Kozarska Dubica",
```

Comment:
These `meso` properties may not be correct, but Who's On First maintains the original source values in source properties. In this case, the [Mesoshapes source](https://github.com/whosonfirst/whosonfirst-sources/blob/main/sources/meso.json) maintains the "Bosanska Dubica" names, so we'll keep them as-is in WOF.

The `name:`, `wof:`, and `label:` property values are fine to update because those are maintained by WOF.

Possible to revert the changes to any source properties like this one?